Makefile

```diff
@@ -27,4 +27,27 @@ all-tests:
 generate build clean:
 	go $@ $(EXTRAGOARGS)
 
-.PHONY: all generate clean build test unit-tests all-tests
+clean:
+	go $@ $(EXTRAGOARGS)
```

Comment:
Since you don't have a variable argument, you can just write `go clean $(EXTRAGOARGS)`.